.github/workflows/lint.yml

```diff
@@ -0,0 +1,14 @@
+name: CI
```

Comment:
Is this file a dup from the other PR?

Reply:
nope. it has a different name and runs a diffrent command.

the reason they are separate files it shows up as separate jobs. 

I think you can merge them to achieve the same thing but it just seemed a bit cleaner to have separate files.